app/src/main/java/com/practicum/playlistmaker/App.kt

```diff
@@ -0,0 +1,36 @@
+package com.practicum.playlistmaker
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+
+const val SHARED_PREFERENCES_SETTINGS = "settings"
+const val PREFERENCES_DARK_THEME = "dark_theme"
+
+class App : Application() {
+    var darkTheme = false
```

Comment:
Переменную лучше сделать приватной, чтобы не было возможности снаружи поменять ей значение, тем самым сломав логику этого класса. Либо можно сделать приватный сеттер:
```suggestion
    var darkTheme = false
        private set
```